hellodagger/src/main/java/com/airbnb/mvrx/helloDagger/HelloFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.airbnb.mvrx.helloDagger
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.airbnb.mvrx.*
+import kotlinx.android.synthetic.main.fragment_hello.*
+
+class HelloFragment: BaseMvRxFragment() {
+
+    private val viewModel: HelloViewModel by fragmentViewModel()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_hello, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
You don't need super for onViewCreated

Reply:
Right, it's an empty method. Thanks, I've fixed this.